.agents/rules/check-rule-bridges.ps1

```
@@ -0,0 +1,106 @@
+    $sourceRuleMd = $source.FullName
+    $codexBridgeRuleMd = Join-Path $codexBridgeRoot $name
+
+    if (-not (Test-Path $codexBridgeRuleMd)) {
+        $errors.Add("Missing Codex bridge rule for '$name'.")
+    }
+
+    $sourceData = Get-RuleData -RulePath $sourceRuleMd
+    $codexData = Get-RuleData -RulePath $codexBridgeRuleMd
+
+    $expectedSourceRule = "../../../.agent/rules/$name"
+    $expectedCodexBridgeRef = "../../../.agents/rules/$name"
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'expectedCodexBridgeRef' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)